src/main/java/address/updater/UpdateManager.java

```diff
@@ -164,6 +171,10 @@ private void checkForUpdate() {
         downloadedVersions.add(latestVersion.get());
     }
 
+    private boolean isJarUpdaterExist() {
```

Comment:
Should be `isJarUpdaterResourceExist` not to confuse with the extracted file. And should return positive value if it exists (negate at the `if` caller).